app/trackedRoute.js

```diff
@@ -13,6 +14,7 @@ export default class TrackedRoute extends React.Component {
 
   render() {
     const { path } = this.props
+    document.title = config.titles[path.split('/')[1]] || 'eLife'
```

Comment:
This is something which could stop the rest of the page rendering if an exception is thrown. I think it's worth adding some error handling/null checks/setting more defaults around `config.titles[path.split('/')[1]]`

Reply:
added error handling